jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l

```diff
@@ -428,9 +618,58 @@ Args:
       (room-light-off :control-switchbot control-switchbot))
     ;; change the inflation_radius
     (restore-params)
+    (send-kitchen-mail)
     (setq success-battery-charging (equal (get-battery-charging-state) :charging))
     (and success-go-to-kitchen success-auto-dock success-battery-charging)))
 
+(defun report-start-go-to-kitchen-state (args)
+  (let ()
+    (report-start-go-to-kitchen-state)
+    (set-alist 'description "キッチンを見に行くよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    t))
+
+
+(defun get-light-on-state (args)
+  "get-light-on-state"
+  (let ((light-on (get-light-on)))
+    (set-alist 'initial-light-on light-on args)
+    (set-alist 'description "電気がついているか確認したよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    light-on))
+
+(defun report-light-on-state (args)
+  "report-light-on-state"
+  (let ()
+    (report-light-on-state)
+    (set-alist 'description "電気がついていたよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    t))
+
+(defun room-light-on-state (args)
+  "room-light-on-state"
+  (let ((control-switchbot (cdr (assoc 'control-switchbot args))))
+    (report-light-off)
+    (room-light-on :control-switchbot control-switchbot)
+    (set-alist 'description "電気をつけたよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    t))
+
+(defun move-to-sink-front-status (args)
+  (let* ((n-trial (cdr (assoc 'n-kitchen-trial args)))
+         (success (move-to-sink-front :n-trial n-trial)))
+    (setf (cdr (assoc 'success-go-to-kitchen args)) success)
+    (set-alist 'description "キッチンに向かったよ" argw)
```

Comment:
If we can undeerstand DESCRIPTION as transition (arrow) of smach state, can we write DESCRIPTION based on the  exit status of each node ? instead of https://github.com/jsk-ros-pkg/jsk_robot/pull/1568/files#diff-db470e8365f9cf73b75d39c7822ddc1513068c80cf3e12dd4a3127cba08d2863R684-R687 ???
```
(if succcess 
    (set-alist 'description "キッチンに向かったよ" argw)
    (set-alist 'description "キッチンに向かったけど，迷子になっちゃった" argw)
)

```
c.f https://github.com/jsk-ros-pkg/jsk_roseus/pull/716#issuecomment-1225047867

Reply:
Yes. 
I use exit status of each node in 1213188